### DISCUSSION TITLE
ULCA Hindi ASR Dataset Corpus

### DISCUSSION THREAD

**Discussion by muni7085** [NVIDIA-NeMo/NeMo#7898] [Q&A]
I working on Hindi ASR system. For that I collecting transcribed Hindi speech data. In this process I found the ULCA ASR Dataset Corpus in GitHub. But the links seems to be broken. I found the NVIDIA RIVA Hindi ASR models are trained on ULCA Hindi ASR Dataset Corpus. Is there any other sources to get this dataset?
Can anyone please help 🙏.

**Comment by 1-800-BAD-CODE** [comment]
I'm unaffiliated with nvidia and that model you mention, but I'm familiar with this data so I'll give you my two cents.

The reason you cannot find ULCA for download is because it contains explicitly-copyrighted materials. The authors presumably do not have permission to distribute the data, and they ghost you if you ask about it (https://github.com/Open-Speech-EkStep/ULCA-asr-dataset-corpus/issues/4).

If you are not already aware of it, you may be interested in the [Shrutilipi dataset](https://ai4bharat.iitm.ac.in/shrutilipi/).

Though Shrutilipi is also collected from broadcasts, it differs from ULCA in these ways:
1. It's collected from a single source (easier to verify whether it's ok for you to use it)
2. It's collected from a government-affiliated source (less likely to bring litigation/take-down notices)
3. It's collected in and distributed from India
4. The authors describe in detail where it came from and how it was collected
5. The authors use an appropriate license (they license the packaging, not the data)

**Reply by muni7085** [reply]
I appreciate your answer. Thank you for your help.
